test/multiverse/suites/active_record_pg/Envfile

```diff
@@ -27,7 +27,7 @@ def gem_list(activerecord_version = nil)
   RB
 end
 
-
+create_gemfiles(ACTIVERECORD_VERSIONS, gem_list)
```

Comment:
The create_gemfiles call is currently missing from dev, but will be addressed in PR https://github.com/newrelic/newrelic-ruby-agent/pull/1268. Needed this update for testing, so it is included here.

Reply:
No longer relevant comment as of PR #1268